omod/src/test/java/org/openmrs/module/coreapps/htmlformentry/EncounterDispositionTagHandlerComponentTest.java

```diff
@@ -192,16 +190,15 @@ public Map<String, Object> getFormEntrySessionAttributes() {
 
             @Override
             public String[] widgetLabels() {
-                return new String[] { "Date:", "Location:", "Provider:", "Disposition:", "Encounter Type:" };
```

Comment:
Could you remind me why we had to drop Encounter Type?

Reply:
I thought at one point it was leading to a test failure.  I think it was a red herring, but the tests pass without it, and it isn't what is being tested, so I just left it out.